TurkishFilter/sections/specific.txt

```diff
@@ -119,6 +119,7 @@ birhayalinpesinde.com##center > a[href*="://bit.ly/"] > img
 memurlar.net##.ColumnContainer > div[style="float:left; width:445px; height:350px"]
 ||memurlar.net/common/widget-image/kitapsec/16.jpg
 memurlar.net##body > #masth.PageWidth[style*="text-align:center!important"]
+||imgyukle.com/*/kNGVnp.gif
```

Comment:
.p-body-inner > .samCodeUnit

Reply:
don't remove existing rule if you will add element hiding. This gif is big.